lib/requestIdleCallbackPolyfill.js

```diff
@@ -0,0 +1,24 @@
+// A poyfill for requestIdleCallback.
+//
+// For: IE 11, Edge, and Firefox
+// Source: https://www.npmjs.com/package/requestidlecallback-polyfill
+// https://developer.mozilla.org/en-US/docs/Web/API/Window/requestIdleCallback
+window.requestIdleCallback =
```

Comment:
The intention would be to add this polyfill to our `polyfill.js`, but that is private and this library is OSS so it felt wrong not to include it here. Given we already have a `polyfill.js` file to load, would there be an advantage to masking this in Dart rather than including it there?

Reply:
We don't really have a standardized `polyfill.js` - a file with that name was added as a stopgap to address a SEV1 [here](https://github.com/Workiva/wdesk_sdk/pull/3054). Ideally this would be moved to a static asset that is loaded by the experience(s) that need it. Or maybe we would do as Rob suggested and use polyfill.io.

Reply:
Given the use of the simpler `new Future.delayed` trick if `requestIdleFrame` isn't around, do you think we could get away with just leaving this polyfill in for OSS users but not using it in Wdesk? With Chrome and Firefox support and the new Edge announcement we may not need to add a polyfill to the code after all.